scripts/kfctl.sh

```diff
@@ -49,13 +49,64 @@ createEnv() {
       echo KUBEFLOW_DOCKER_REGISTRY=registry.aliyuncs.com >> ${ENV_FILE}
       ;;
     gcp)
-      PROJECT=${PROJECT:-$(gcloud config get-value project 2>/dev/null)}
+      while [[ $# -gt 0 ]]; do
```

Comment:
Thanks for doing this.

Do you think we could support the syntax?
--name value
as well as
--name=value

That could be a separate PR.

Reply:
We can @jlewi , I just have to put more logic to cover both scenarios 🙂 
Feel free to file the ticket and describe it the way we want it to be done and assign it to me and I'll pick it up after this one.
But first let's get the tests fixed🤞